upload-api/functions/ucan-invocation-router.js

```diff
@@ -86,6 +86,9 @@ export async function ucanInvocationRouter (request) {
     SUBSCRIPTION_TABLE_NAME: subscriptionTableName = '',
     DELEGATION_TABLE_NAME: delegationTableName = '',
     DELEGATION_BUCKET_NAME: delegationBucketName = '',
+    R2_DELEGATION_BUCKET_ENDPOINT: r2DelegationBucketEndpoint = '',
+    R2_DELEGATION_BUCKET_ACCESS_KEY_ID: r2DelegationBucketAccessKeyId = '',
+    R2_DELEGATION_BUCKET_SECRET_ACCESS_KEY: r2DelegationBucketSecretAccessKey = '',
```

Comment:
We should also have the name, like we do for replicator https://github.com/web3-storage/w3infra/#r2_carpark_bucket_name (even if it currently the same)